manual.adoc

```diff
@@ -280,7 +280,7 @@ Also see the https://emacs-lsp.github.io/lsp-mode/page/lsp-rust-analyzer/[rust-a
 
 Note the excellent https://robert.kra.hn/posts/2021-02-07_rust-with-emacs/[guide] from https://github.com/rksm[@rksm] on how to set-up Emacs for Rust development with LSP mode and several other packages.
 
-=== Vim/NeoVim
+=== Vim/Neovim
```

Comment:
Can you move this to https://github.com/rust-lang/rust-analyzer/blob/master/docs/user/manual.adoc?

Reply:
Should I discard the changes in manual.adoc and send the same pull request to Rust-lang/Rust-analyzer?

Reply:
Then manual is in the main repo, the changelog is here.